docs/source/architecture_decisions.md

```diff
@@ -4,23 +4,23 @@ This is a collection of records for architecturally significant decisions.
 
 ### URL Patterns
 
-We follow a very basic yet strong convention for URL so that our rest APIs are properly namespaced. First of all, we rely heavily on HTTP verbs to perform **CRUD** actions.
+We follow a very basic ,yet strong convention for URL so that our rest API's are properly namespaced. First of all, we rely heavily on HTTP verbs to perform **CRUD** actions.
```

Comment:
Please remove the space between `basic` & `,`. Add it between `,` and `yet` like `basic, yet`